### ISSUE TITLE
occurred is misspelled several times in en_GB.lib.joomla.ini file

### ISSUE THREAD

**Issue by Stevec4** [joomla/joomla-cms#11038]
#### Steps to reproduce the issue
occurred is misspelled  occured   on lines 631, 633, 634, 635,

Corrected as below
```
JLIB_JS_AJAX_ERROR_CONNECTION_ABORT="A connection abort has occurred while fetching the JSON data."
JLIB_JS_AJAX_ERROR_NO_CONTENT="No content was returned."
JLIB_JS_AJAX_ERROR_OTHER="An error has occurred while fetching the JSON data: HTTP %s status code."
style="_QQ_"colour:inherit;white-space:pre-wrap;padding:0;margin:0;border:0;background:inherit;"_QQ_">%s</code>"
JLIB_JS_AJAX_ERROR_TIMEOUT="A timeout has occurred while fetching the JSON data."


```

**Comment by brianteeman** [comment]
Closed as we have a PR #11039

**Comment by Stevec4** [comment]
Thanks Brian

Steve
